src/db.rs

```diff
@@ -339,6 +362,20 @@ impl DbInner {
 
         Ok(())
     }
+
+    async fn load_sst_iters(
```

Comment:
Nit: Can you define this as a nested/inner function inside `replay_wal`? It's pretty specific to that chunk of logic.

Reply:
done